nova/bigvm/manager.py

```diff
@@ -331,6 +341,8 @@ def _get_providers(self, context):
                                    - memory_mb_inventory['reserved'])
                 memory_mb_used_percent = (usages[MEMORY_MB]
                                           / float(memory_mb_total) * 100)
+                memory_mb_reserved_percent = (memory_mb_inventory['reserved']
```

Comment:
Now I notice that `'reserved'` does not mean what I thought it meant. So do we have to gather all VMs, check if they have memory reservation on and add it ourselves?

Reply:
https://docs.openstack.org/api-ref/placement/?expanded=list-resource-provider-inventories-detail#resource-provider-inventories